package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -840,7 +820,7 @@ def timeseries(self):
         --------
         :attr:`table` : structured array of the data
         '''
-        return super().timeseries
+        return super(WaterBridgeAnalysis, self).timeseries
```

Comment:
So `super()` on its own did not work? Or is it a Py2/3 thing?

Reply:
@orbeckst I have tested super() in py3 and it is completely fine. However, py2 complains about lacking position argument.